include/file.h

```diff
@@ -0,0 +1,16 @@
+#ifndef INCLUDE_FILE_H_
+#define INCLUDE_FILE_H_
```

Comment:
The variable naming scheme here is not standardized across source files. Please use `_FULLTEXT_NAME_H_` across the board (that is, this would be `_FULLTEXT_FILE_H_`).